addons/binding/org.openhab.binding.tplinksmarthome/src/main/java/org/openhab/binding/tplinksmarthome/TPLinkSmartHomeBindingConstants.java

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.tplinksmarthome;
+
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+import com.google.common.collect.ImmutableSet;
+
+/**
+ * This class defines common constants, which are used across the whole binding.
+ *
+ * @author Christian Fischer - Initial contribution
+ * @author Hilbrand Bouwkamp - Added channel and property keys
+ */
+@NonNullByDefault
+public final class TPLinkSmartHomeBindingConstants {
+
+    public static final String BINDING_ID = "tplinksmarthome";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_HS100 = new ThingTypeUID(BINDING_ID, "hs100");
+    public static final ThingTypeUID THING_TYPE_HS105 = new ThingTypeUID(BINDING_ID, "hs105");
+    public static final ThingTypeUID THING_TYPE_HS110 = new ThingTypeUID(BINDING_ID, "hs110");
+    public static final ThingTypeUID THING_TYPE_HS200 = new ThingTypeUID(BINDING_ID, "hs200");
+    public static final ThingTypeUID THING_TYPE_LB100 = new ThingTypeUID(BINDING_ID, "lb100");
+    public static final ThingTypeUID THING_TYPE_LB110 = new ThingTypeUID(BINDING_ID, "lb110");
+    public static final ThingTypeUID THING_TYPE_LB120 = new ThingTypeUID(BINDING_ID, "lb120");
+    public static final ThingTypeUID THING_TYPE_LB130 = new ThingTypeUID(BINDING_ID, "lb130");
+    public static final ThingTypeUID THING_TYPE_RE270K = new ThingTypeUID(BINDING_ID, "re270");
+    public static final ThingTypeUID THING_TYPE_RE370K = new ThingTypeUID(BINDING_ID, "re370");
+
+    /**
+     * All supported Smart Devices.
+     */
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = ImmutableSet.of(THING_TYPE_HS100, THING_TYPE_HS105,
+            THING_TYPE_HS110, THING_TYPE_HS200, THING_TYPE_LB100, THING_TYPE_LB110, THING_TYPE_LB120, THING_TYPE_LB130,
+            THING_TYPE_RE270K, THING_TYPE_RE370K);
+    /**
+     * All Smart Home on/off devices, like switch and plugs.
+     */
+    public static final Set<ThingTypeUID> SWITCH_THING_TYPES = ImmutableSet.of(THING_TYPE_HS100, THING_TYPE_HS105,
+            THING_TYPE_HS200, THING_TYPE_RE270K, THING_TYPE_RE370K);
+    /**
+     * All Smart Home on/off devices that also can monitor energy. Only HS110 supports this.
+     */
+    public static final Set<ThingTypeUID> SWITCH_ENERY_THING_TYPES = ImmutableSet.of(THING_TYPE_HS110);
+
+    public static final Set<ThingTypeUID> BULB_THING_TYPES = ImmutableSet.of(THING_TYPE_LB100, THING_TYPE_LB110,
+            THING_TYPE_LB120, THING_TYPE_LB130);
+
+    // List of default refresh rates in seconds.
```

Comment:
This comment feels not really attached to the two fields below it, why not make the `REFRESH_SMART_PLUG` a `Duration` or some other things that includes a unit. Or the cheaper refactoring `REFRESH_SMART_PLUG_SEC(ONDS)`

Because its harder to check 

```java
//  refresh rates in seconds.
public static final int REFRESH_SMART_PLUG = 30;
```

```java
scheduler.scheduleWithFixedDelay(this:doSomething, 0, REFRESH_SMART_PLUG, TimeUnit.MSEC);
```

Didn't a space shuttle exploded because of this 😜 

PS: its not critical but no comments on a PR is boring 😉

Reply:
There was even a simpler fix: remove them, they were not used at all anymore ;-) 
I think you are referring to the Ariana 5 rocket that crashed due to an overflow error in an unused module.